src/odf/metadata/src/dtos/dtos_enum_flags_generated.rs

```diff
@@ -1,63 +0,0 @@
-// Copyright Kamu Data, Inc. and contributors. All rights reserved.
-//
-// Use of this software is governed by the Business Source License
-// included in the LICENSE file.
-//
-// As of the Change Date specified in that file, in accordance with
-// the Business Source License, use of this software will be governed
-// by the Apache License, Version 2.0.
-
-////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-// WARNING: This file is auto-generated from Open Data Fabric Schemas
-// See: http://opendatafabric.org/
-////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-
-use bitflags::bitflags;
-
-use crate::MetadataEvent;
-
-////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-
-////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-// MetadataEvent
-// https://github.com/kamu-data/open-data-fabric/blob/master/open-data-fabric.md#metadataevent-schema
-////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-
-bitflags! {
```

Comment:
Note: I combined bitflags generation with the DTO codegen, so it's in a different file now